doc/release-notes/2.6.2.md

```diff
@@ -456,6 +456,11 @@ Language
     ... | y = recompute y
   ```
 
+* The cubical interval `I` now belongs to its own sort, `IUniv`, rather
+  than `SSet`. For `J : ISet` and `A : J → Set l`, we have
+  `(j : J) → A : Set l`, that is, the type of functions from a type in `ISet`
+  to a fibrant type is fibrant.
+
 Builtins
```

Comment:
I think this should go into `CHANGELOG.md`, since  2.6.2 is already branched off and feature-frozen. Unless we have a strong reason to merge this into 2.6.2.

Reply:
Alas, I don't think my reasons are strong enough 😅 I'll move it.